multimedia.py

```diff
@@ -35,15 +47,15 @@
 
 
 
-@app.route('/')
+
```

Comment:
wait don't we need @app.route('/')  ?

Reply:
if you look on line 58 of the multimedia.py. The login page become the new @app.route("?")